test/acceptance/test_cli.py

```diff
@@ -125,6 +127,9 @@ def test_exec_vint_with_verbose_flag(self):
         self.assertRegex(got_output, expected_output_pattern)
 
 
+    @pytest.mark.skipif(
+        os.environ.get('GITHUB_ACTIONS') == 'true',
+        reason='commands are not run in interactive shell on GitHub Actions')
     def test_exec_vint_with_color_flag(self):
```

Comment:
I suggest passing `TERM=xterm` to the subprocess (unconditionally) via `env` with `subprocess.check_output`.